src/components/Shared/Filters/NullState/NullState.tsx

```diff
@@ -88,19 +81,20 @@ const NullState: React.FC<Props> = ({
   ),
 }: Props) => {
   const { t } = useTranslation();
+  const { searchTerm, setActiveFilters, isFiltered } = useUrlFilters();
 
   return (
     <NullStateBox data-testid={`${page}-null-state`}>
       <Icon
         path={page === 'contact' ? mdiHome : mdiFormatListBulleted}
         size={1.5}
       />
-      {filtered ? (
+      {isFiltered || !!searchTerm ? (
```

Comment:
You don't really need the double-negation here.

```suggestion
      {isFiltered || searchTerm ? (
```